.travis.yml

```diff
@@ -10,8 +10,9 @@ install:
   # issues with releases
   - pip install --upgrade setuptools pip
   - pip install -r dev-requirements.txt
-  - python setup.py bdist_wheel
-  - pip install dist/*.whl
+  - pip install -e .
+ # - python setup.py bdist_wheel
```

Comment:
IIRC this was added to catch mistakes in `MANIFEST.in` that we'd only noticed after making a release. So if this saves us time in each build we should find a replacement for this check.

Reply:
MANIFEST.in doesn't affect bdist_wheel, only sdists. But this does test that we are installing the files we need, where `pip install -e` does not. A `pip install .` should cover the same, though.